### PR TITLE
Update apiVersion to operators.coreos.com/v1alpha1

### LINKED ISSUE #21: Error while deploying the operator via OLM
I was following the Getting Started Guide's [Deploy the Operator](https://github.com/operator-framework/getting-started#manage-the-operator-using-the-operator-lifecycle-manager) section and tried to install memcachedoperator csv after deploying OLM, but I got this error:
```
$ kubectl apply -f memcachedoperator.0.0.1.csv.yaml
error: unable to recognize "memcachedoperator.0.0.1.csv.yaml": no matches for kind "ClusterServiceVersion-v1" in version "app.coreos.com/v1alpha1"
```
I also tried removing `-v1` from `ClusterServiceVersion-v1` in the file, but I get the same error:
```
$ kubectl apply -f memcachedoperator.0.0.1.csv.yaml 
error: unable to recognize "memcachedoperator.0.0.1.csv.yaml": no matches for kind "ClusterServiceVersion" in version "app.coreos.com/v1alpha1"
``` 
I'm not aware of much of the internals of OLM at the moment. Some hint would be great.

### PR DESCRIPTION
**Pull request by darkowlzz**
csv is defined under `operators.coreos.com` group in the latest version
of OLM.

Fixes #21

### CODE REVIEW

File: memcachedoperator.0.0.2.csv.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # This file defines the ClusterServiceVersion (CSV) to tell the catalog how to display, create and
 # manage the application as a whole. If changes are made to the CRD for this application kind,
 # make sure to replace those references below as well.
-apiVersion: app.coreos.com/v1alpha1
+apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion-v1
```
Comment (LiliC) [inline memcachedoperator.0.0.2.csv.yaml]: The `-v1` also needs to be removed.

File: memcachedoperator.0.0.1.csv.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # This file defines the ClusterServiceVersion (CSV) to tell the catalog how to display, create and
 # manage the application as a whole. If changes are made to the CRD for this application kind,
 # make sure to replace those references below as well.
-apiVersion: app.coreos.com/v1alpha1
+apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion-v1
```
Comment (LiliC) [inline memcachedoperator.0.0.1.csv.yaml]: The `-v1` also needs to be removed.

**Comment by LiliC** [comment]
Also I noticed in the [README](https://github.com/operator-framework/getting-started/blob/master/README.md#deploy-the-operator) there is a query that should be removed that includes the `v1s`. 

```
$ kubectl get ClusterServiceVersion memcachedoperator.v0.0.1 -o json | jq '.status'
```

**Comment by darkowlzz** [comment]
oops! 😅 I thought I did remove `-v1` from there.
Added the suggested changes in the same commit.
Thanks!

**Pull request merged**